### ISSUE TITLE
Topic: filter by extras value

### ISSUE THREAD

**Issue by abulte** [ecolabdata/ecospheres-front#51]
In order to display all ecospheres-related topics, we need a way to filter the topics list by a value of a particular extra. 

GET `/topics?extras__is_ecospheres=True`

**Comment by abulte** [comment]
@maudetes yes certainement plus simple avec les tags, qui sont "contrôlés" sur les topics. On est d'accord que c'est pas encore possible ?

**Comment by maudetes** [comment]
yes, c'est à faire !

**Comment by streino** [comment]
@maudetes Une idée de quand le filtre sur tag pourrait être mis en place de votre côté ?
Le workaround a pas l'air convaincant à ce stade : https://github.com/ecolabdata/ecospheres-front/pull/89
On va se retrouver coincé car si on pousse trop de datasets dans notre univers, notre page d'affichage/édition bouquets va ramer sérieusement.

**Comment by maudetes** [comment]
C'est en cours de traitement dans https://github.com/opendatateam/udata/pull/2904, ça devrait arriver vite
